implementation/src/main/java/io/smallrye/restclient/RestClientBuilderImpl.java

```diff
@@ -272,32 +270,33 @@ public Configuration getConfiguration() {
     @Override
     public RestClientBuilder property(String name, Object value) {
         if (name.startsWith(RESTEASY_PROPERTY_PREFIX)) {
-            // Allows to configure some of the ResteasyClientBuilder delegate properties
+            // Makes it possible to configure some of the ResteasyClientBuilder delegate properties
             String builderMethodName = name.substring(RESTEASY_PROPERTY_PREFIX.length());
+            Method builderMethod = Arrays.stream(ResteasyClientBuilder.class.getMethods())
```

Comment:
it's more readable with `findFirst` and `orElse` in their own lines ;)

Reply:
Is it better now? ;-)